types/algoliasearch.js

```diff
@@ -0,0 +1 @@
+// fake file to work with the types
```

Comment:
Can you elaborate a bit in this comment? It's not fully clear when reading it out of context.

Reply:
```suggestion
// fake file to allow `export * from 'algoliasearch-helper/types/algoliasearch'`
```

is that good?